### PR TITLE
Update menu header to use semantic ui

### PR DESCRIPTION
**Pull request by dreamycodex**
Please review. Thank you.

### CODE REVIEW

File: client/src/Routes.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import Home from "./components/Home";
+import About from "./components/About";
+import Status from "./components/Status";
+
+const Routes = () => {
+  return (
+    <Router>
+      <div>
+        <List link>
+          <List.Item to="/" active as={Link}>
```
Comment (llh1) [inline client/src/Routes.js]: The active props should be set on the active link only. So if the user is currently on the home page, the home link is active. If the user is on the /about page, the About link should be active instead and so on.

**Review by llh1 — DISMISSED** [review]
Looks good, well done 👍

**Review by DwinaTech — DISMISSED** [review]
LGTM 👍🏿

**Review by DwinaTech — APPROVED** [review]
Good fix 👍🏿

**Pull request merged**